### ISSUE TITLE
Port tests to new sequential setup

### ISSUE THREAD

**Issue by jadephilipoom** [project-oak/silveroak#565]
See #561 -- code under `tests` should be ported to use the new `Circuit` inductive so that the old `seqType`-based `Cava` instance can be eventually removed.

**Comment by smore-lore** [comment]
After https://github.com/project-oak/silveroak/pull/589

I think we only have these left:
acorn-examples/IncrDecr.v:24:Require Import Cava.Acorn.CombinationalProperties.
cava/Cava/Acorn/SequentialProperties.v:31:Require Import Cava.Acorn.CombinationalProperties.

So I think we can also remove/rename these as part of this task.

**Comment by jadephilipoom** [comment]
You're correct, once this is finished we can do some cleanup. That cleanup, including the removing/renaming, is already tracked in a separate issue, though: #567
